install/disconnected_install.adoc

```diff
@@ -169,15 +170,16 @@ rhel-7-server-extras-rpms \
 rhel-7-server-ansible-2.6-rpms \
 rhel-7-server-ose-3.10-rpms
 do
-  reposync --gpgcheck -lm --repoid=${repo} --download_path=/path/to/repos
-  createrepo -v </path/to/repos/>${repo} -o </path/to/repos/>${repo}
+  reposync --gpgcheck -lm --repoid=${repo} --download_path=</path/to/repos> <1>
+  createrepo -v </path/to/repos/>${repo} -o </path/to/repos/>${repo} <1>
 done
 ----
+<1> Provide the path to the directory you created.
 
 [[disconnected-syncing-images]]
-=== Syncing Images
+=== Obtaining images
 
-To sync the container images:
+Pull the required container images:
```

Comment:
Currently etcd image tag is hard-code to "3.2.22", and etcd image should be treated separately different with other ose image.
So "docker pull registry.access.redhat.com/rhel7/etcd" should be "docker pull registry.access.redhat.com/rhel7/etcd:3.2.22", and when sync this image to private registry, we also need the same image tag.